rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_subscriber_info.hpp

```diff
@@ -142,7 +131,7 @@ class SubListener : public EventListenerInterface, public eprosima::fastrtps::Su
   }
 
   void
-  data_taken(eprosima::fastrtps::Subscriber * sub)
+  update_unread_count(eprosima::fastrtps::Subscriber * sub)
```

Comment:
Is it possible to make this change in an ABI compatible way (e.g. leave `data_taken` and call `update_unread_count` from it, or avoid the rename entirely)?

Reply:
Forwarding the request to @iuhilnehc-ynos @fujitatomoya @MiguelCompany which iterated on the original patch.

Reply:
Since avoiding the renaming completely I reverted it in 648f9b0d159590c4a70adfeeb5f8aaf7bead24ca.